Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -559,7 +440,27 @@ def get_incident_command(client, incident_id):
 def get_incident(client, incident_id, content_format=False):
     url = '/incidents/' + str(incident_id)
     if content_format:
-        url += '?text_content_output_format=objects_convert_html'
+        # url += '?text_content_output_format=objects_convert_html'
+        url += '?handle_format=names'
+
+        # enhancement (#13657) commit 7521e4f411de7eeebbdcfeae015cad44512cd528
+        # is invalid and creates a fault with
+        # all fields with enums (i.e. dropdown lists):
```

Comment:
@MosheEichler this was your commit, do you have any suggestions regarding this comment?

Reply:
@neilw73 lets make the changes simply, and ignore the html wrapping for now